From b28d431ca95b4c672be8bbbfc7f9b1e57641759e Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Fri, 10 Feb 2023 14:27:46 +0800
Subject: [PATCH] aptiv-cvc-fl: add cmd to initialize RTL9010 phys in PFE_INIT_CMD

The rtl9010 phys for pfe0/2 need to be configured before linux kernel
boots.

This patch comes from:
meta-cvc-fl/recipes-bsp/u-boot/u-boot-s32/0001-ESL-426_CVC_FL_Uboot_configuration_and_cleanup.patch

Upstream-Status: Inappropriate [WR Linux specific change]

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 include/configs/s32g3xxaevb.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/configs/s32g3xxaevb.h b/include/configs/s32g3xxaevb.h
index b87c750f63..640807abbc 100644
--- a/include/configs/s32g3xxaevb.h
+++ b/include/configs/s32g3xxaevb.h
@@ -17,6 +17,9 @@
 #ifdef CONFIG_FSL_PFENG
 #  define PFENG_MODE			"enable,sgmii,sgmii,sgmii"
 #  define PFENG_EMAC			"1"
+#  define PFE_INIT_CMD			"mii device pfeng_emac_0; mii write 1 1b 0xCC04; mii write 1 1c 0x7180;" \
+								"mii device pfeng_emac_2; mii write 1 1b 0xCC04; mii write 1 1c 0x7180;" \
+								"pfeng stop;"
 #endif
 
 #else
-- 
2.36.1

